code/game/machinery/civilian_bounties.dm

```diff
@@ -136,6 +136,7 @@
 		//Pay for the bounty with the ID's department funds.
 		status_report += "Bounty completed! Please give your bounty cube to cargo for your automated payout shortly."
```

Comment:
```suggestion
		status_report += " Bounty completed! Please give your bounty cube to cargo for your automated payout shortly."
```
[This isn't you but I don't know how long it's gone unnoticed that it's missing a space so I'm going to be evil]